bears/general/InvalidLinkBear.py

```diff
@@ -79,14 +81,14 @@ def find_links_in_file(file, timeout, link_ignore_regex):
             match = regex.search(line)
             if match:
                 link = match.group()
-                if not link_ignore_regex.search(link):
+                if not (link_ignore_regex.search(link) or (fnmatch(link, links_to_ignore))):
```

Comment:
Go through `https://coala.io/commit`.
Always check the things in gitmate-bot's comment.